### PR TITLE
More coverity fixes

### PR DESCRIPTION
**Pull request by harveyj1953**

### CODE REVIEW

**Comment by vgvassilev** [comment]
@phsft-bot build!

File: hist/hist/src/TFormula_v5.cxx [hunk 1]
```diff
@@ -152,7 +152,7 @@ TFormula::TFormula(): TNamed()
    fOperOptimized  = 0;
    fOperOffset     = 0;
    fPredefined     = 0;
-   fOptimal        = (ROOT::v5::TFormulaPrimitive::TFuncG)&TFormula::EvalParOld;
```
Comment (Axel-Naumann) [inline hist/hist/src/TFormula_v5.cxx]: Why do we have these changes? Are they only whitespace changes?

File: hist/hist/src/TFormula_v5.cxx [hunk 2]
```diff
@@ -2685,10 +2686,10 @@ Double_t TFormula::EvalParOld(const Double_t *x, const Double_t *uparams)
 {
    Int_t i,j;
    // coverity[uninit] the tab value of tab is guaranteed to be set properly by the control flow.
-   Double_t tab[kMAXFOUND];
-   const char *stringStack[gMAXSTRINGFOUND];
+   Double_t tab[kMAXFOUND] = {0};
```
Comment (Axel-Naumann) [inline hist/hist/src/TFormula_v5.cxx]: Why do we have comment *and* initialization? I'd assume that one of them could go away?

Comment (Axel-Naumann) [inline hist/hist/src/TFormula_v5.cxx]: These initializations cost time. @lmoneta - are they actually needed or is Coverity incorrect here? The comment seems to claim Coverity is wrong.

Comment (lmoneta) [inline hist/hist/src/TFormula_v5.cxx]: I think @pcanal added these comments, so he might know. 
I think Axel is wight and we should ignore Coverity here

Comment (pcanal) [inline hist/hist/src/TFormula_v5.cxx]: The last time I checked, the control flow insure it was initialized before use.  Given that this a deprecated class, I would not change.

Albeit, it looks like the comment needs to be updated for the current version of coverity and/or the report be flagged as false positive.

**Comment by vgvassilev** [comment]
@phsft-bot build!

**Comment by vgvassilev** [comment]
@Axel-Naumann, isn't the reference to `v5` stuff out of the scope of this PR? How we can proceed further?

**Comment by pcanal** [comment]
@vgvassilev what do you mean? This PR is proposing changes to formula_v5 and @axel ask clarification on them ....  [note: this source file is used to read 'old' ROOT files]

**Comment by amadio** [comment]
What should we do with this PR? Should I cherry-pick the approved changes and merge?

**Comment by Axel-Naumann** [comment]
I'd say merge as is. The whitespace changes in TFormula_v5 are just a tiny annoyance compared to leaving the members uninitialized.

**Comment by etejedor** [comment]
Should we then go ahead with the merge of this PR?

**Pull request merged**